bin/ds9_imgproc_wrapper

```diff
@@ -178,9 +178,14 @@ elif "wavdetect" == toolname:
     tool.imagefile=tool.outfile+"_reconimg"
     tool.defnbkgfile=tool.outfile+"_nbkg"
     tool.interdir=os.environ["DAX_OUTDIR"]
+    tool.expfile=sys.argv[4][1:].strip('"') # strip off leading 'e' 
+    tool.psffile=sys.argv[5][1:].strip('"') # strip off leading 'p'
 elif "celldetect" == toolname:
     output_fn = send_output_region_to_ds9
     tool.fixedcell=int(sys.argv[3])    
+    tool.expstk=sys.argv[4][1:].strip('"') # strip off leading 'e' 
+    tool.psffile=sys.argv[5][1:].strip('"') # strip off leading 'p'
+    tool.thresh=float(sys.argv[6])
```

Comment:
So, making the user pre-create these images is the simplest thing to do, but I wonder if it would make sense (for the dax-like workflow) to allow the task to create the exposure and/or psffiles on the fly? It would obviously complicate the interface (as users would want one of three options: 1) no file, 2) use a file, 3) auto-generate the file) and how to specify the needed parameters for the tools. As well as the life of whoever has to implement this.

Perhaps an enhancement for a future release?

Reply:
The problem is speed; exposure maps take too long at descent resolution to be done via dax.

I've asked Bill to give ds9 a 'File Open' dialog in the parameter selection gui (akin to what peg does).  That will help.

Reply:
It may also be useful for `fluximage` to put the filenames of the expmap and psfmap (if any) into header keywords of the counts image and fluxed image.

Reply:
So then we'd also want to "invent" a syntax for these parameter values to say "use this keyword from the input file name" in a similar manner to the parameter libraries redirect/whatever functionality? Or would you just automatically use them if found (and the user didn't specify these arguments)?

Reply:
Can you open an issue about coming up with some sensible keywords we can use for this?